tools/lib/provision.py

```diff
@@ -45,6 +45,8 @@
 # TODO: De-duplicate this with emoji_dump.py
 EMOJI_CACHE_PATH = "/srv/zulip-emoji-cache"
 is_travis = 'TRAVIS' in os.environ
+is_circleci = 'CIRCLECI' in os.environ
```

Comment:
So the thing that worries me a bit about a solution like this is, where else are we checking for `is_travis`? And which of those places also need to change to include `is_circleci` as well?

Searching this file, there are two more of them:
* `EMOJI_CACHE_PATH`, just below
* `setup_venvs`

The first one is clearly Travis-specific. The other one is not so clear -- you have to go refer to another file to find out. And that code in turn could grow more things it does with that information, without anyone looking back at this file and the `is_circleci` case.

Fortunately, `setup_venvs` doesn't *actually* do much with this.

So can you add a commit before this one to cut that off first?
* Move the little bit of `is_travis` logic in `setup_venvs.py` to its caller, here
* Remove the `is_travis` parameter from `setup_venvs.main`